### DISCUSSION TITLE
Add: Git Demonstration reopen button

### DISCUSSION THREAD

**Discussion by tomi-neuron** [pcottle/learnGitBranching#1148] [Ideas]
Amazing tool thank you very much! However I find myself having to go back to the Git Demonstration window often, to fully understand and apply the stuff explained there, but there is no easy way to do so. I have to reload the page and reopen the given session again in order to have a look at the short "class" on the present commands. Could you add a button on the bottom left alongside the others, to reopen the Git Description window again?

**Comment by pcottle** [comment]
<img width="440" alt="Screenshot 2024-05-27 at 9 23 10 PM" src="https://github.com/pcottle/learnGitBranching/assets/1135007/d4b99715-010a-4ea4-b92e-3d249a4bf905">
Just try typing in `help level` to open up the level dialog again!
